behavior_metrics/brains/f1/brain_f1_torch.py

```diff
@@ -110,7 +110,7 @@ def execute(self):
             with torch.no_grad():
                 image = self.transformations(img).unsqueeze(0)
                 image = FLOAT(image).to(self.device)
-                prediction = self.net(image).numpy()
+                prediction = self.net(image).cpu().numpy() if self.gpu_inference else self.net(image).numpy() 
             self.inference_times.append(time.time() - start_time)
```

Comment:
The current noetic-devel branch and this branch have different lines that have been added in the last weeks:

https://github.com/JdeRobot/BehaviorMetrics/blob/b446b4c891c78707b5504ecc7f17284e1b1a6b3b/behavior_metrics/brains/f1/brain_f1_torch.py#L117

Could you update this file with the latest changes and test that everything is working correctly?